examples/eks_managed_node_group/main.tf

```diff
@@ -10,13 +10,13 @@ provider "kubernetes" {
     api_version = "client.authentication.k8s.io/v1beta1"
     command     = "aws"
     # This requires the awscli to be installed locally where Terraform is executed
-    args = ["eks", "get-token", "--cluster-name", module.eks.cluster_id]
+    args = ["eks", "get-token", "--cluster-name", module.eks.cluster_name]
```

Comment:
what's the reasoning for this change?

Reply:
Seems more correct to pass in the `module.eks.cluster_name` instead of `cluster_id` attribute to `--cluster-name`, no?

Reply:
see the PR changes for background and reasoning. more context can be found here https://github.com/hashicorp/terraform-provider-aws/issues/27560

Reply:
Thanks for explaining. Will modify internally to match